### ISSUE TITLE
group page fields

### ISSUE THREAD

**Issue by dogfrogfog** [focusreactive/cms-kit#47]
group and fold header + sections + footer

so editor see title and url first, fill that, and only after, onboards for visual components
![Oct 9 Screenshot for CMS-KIT](https://github.com/user-attachments/assets/d7fe04e0-063b-4aad-94fb-48c91e0740c7)

**Comment by dogfrogfog** [comment]
already done
